### PR TITLE
Fix nextclade coreutils dependency

### PR DESCRIPTION
**Pull request by mvdbeek**
Without a version we can't use the container.

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/nextclade/nextclade.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     </macros>
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">nextclade</requirement>
-        <requirement type="package">coreutils</requirement>
+        <requirement type="package" version="8.31">coreutils</requirement>
```
Comment (mvdbeek) [inline tools/nextclade/nextclade.xml:9]: Do we even need coreutils ? Is that for the tail command ? The busybox image supports this, so I don't think that's necessary ?

**Comment by bernt-matthias** [comment]
Added here https://github.com/galaxyproject/tools-iuc/pull/3884. @pvanheus can you recall why this has been added?

**Comment by wm75** [comment]
https://github.com/pvanheus/tools-iuc/blob/df6a68b71a4c0af77a960c193850e1ea28b69b47/tools/nextclade/nextclade.xml#L13

`dirname` is from coreutils too, isn't it?

**Review by bernt-matthias — APPROVED** [review]

**Comment by bgruening** [comment]
@bernt-matthias, please feel encouraged to merge PRs :)

**Pull request merged**